%YAML 1.0
---
image_width: 640
image_height: 480
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [
      288.12815904373724, 0.000000, 320.5,
      0.000000, 288.12815904373724, 240.5,
      0.000000, 0.000000, 1.000000 ]
distortion_coefficients: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [0.0, 0.0, 0.0, 0.0, 0.000000]